src/server/scripts/Northrend/CrusadersColiseum/TrialOfTheCrusader/instance_trial_of_the_crusader.cpp

```diff
@@ -230,6 +264,8 @@ class instance_trial_of_the_crusader : public InstanceMapScript
                     break;
                 case GO_ARGENT_COLISEUM_FLOOR:
                     GO_FloorGUID = go->GetGUID();
+                    if (InstanceProgress == INSTANCE_PROGRESS_ANUB_ARAK)
+						go->SetDestructibleState(GO_DESTRUCTIBLE_DAMAGED);
```

Comment:
Please set your programming environment to use 4 spaces instead of tabs, you used tabs everywhere